### DISCUSSION TITLE
Can't upload when this happen

### DISCUSSION THREAD

**Discussion by wachalala** [Natizyskunk/vscode-sftp#285] [General]
Hello,

Thank you for the plugin.
I have just a little issue with it.
Sometimes the loading icon near SFTP in the taskbar keep spining and i have this erreur message : 

> 
![image](https://user-images.githubusercontent.com/34910892/209531958-2675bfae-da49-44c3-8548-103d2110ef98.png)

When it's happen i can't upload anything anymore.
Someone know how i can resolve this problem ?

**Comment by Natizyskunk** [comment]
It's a common known issue. Please refer to he [issues](https://github.com/Natizyskunk/vscode-sftp/issues) tab as you'll find other people with the same problem that have found some workaround.
